ssl/statem/statem_clnt.c

```diff
@@ -3495,37 +3437,6 @@ int tls_client_key_exchange_post_work(SSL_CONNECTION *s)
     pms = NULL;
     pmslen = 0;
```

Comment:
Largely unrelated to the `sctp` diff: `OPENSSL_clear_free()` is only ever called if `pms == NULL`. This function could be a lot cleaner and simpler.